gecko/core/telemetry.py

```
@@ -329,10 +355,17 @@ def get_current_span(self) -> Optional[Span]:
 
 
 def get_telemetry() -> GeckoTelemetry:
-    """获取全局遥测实例"""
+    """
+    获取全局遥测实例。
+    
+    注意：此函数返回的实例会在首次调用时自动初始化（setup）。
```

Ruff (0.14.7)

Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)